javascript/src/stories/components/ChannelMessages/MessageListView.stories.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+
+import MessageListView from "main/components/ChannelMessages/MessageListView";
+
+
+export default {
+  title: 'components/ChannelMessages/MessageListView',
+  component: MessageListView
+};
+
+const Template = (args) => <MessageListView {...args}/>;
+
+export const WithSearchBar = Template.bind({});
+WithSearchBar.args = {
+  messages: [{
+    "type": "message",
+    "subtype": "channel_join",
+    "ts": "1594143066.000200",
+    "user": "U017218J9B3",
+    "text": "<@U017218J9B3> has joined the channel",
+    "channel": "section-6pm",
+    "user_profile": {
+      "real_name": "Test Person"
+    }
+  },
+  {
+    "type": "message",
+    "subtype": "channel_join",
+    "ts": "1594143066.000200",
+    "user": "U017218J9B3",
+    "text": "Office hours at <https://ucsb.zoom.us/j/89220034995?pwd=VTlHNXJpTVgrSEs5QUtlMDdqMC9wQT09>",
+    "channel": "section-6pm",
+    "user_profile": {
+      "real_name": "Test Person"
+    }
+  }],
+};
+
```

Comment:
Rather than hard coding these messages here, it would be better practice to put these into a fixtures file.

See: https://github.com/ucsb-cs156-s21/STARTER-team02/tree/main/javascript/src/fixtures for examples, and this file for an example of using the fixtures in a story:

https://raw.githubusercontent.com/ucsb-cs156-s21/STARTER-team02/main/javascript/src/stories/components/Earthquakes/EarthquakesList.stories.mdx

However, this is ok for now. I suggest adding another issue to refactor this after it is merged; that issue would be worth an extra 5 points.